lib/web_apis/api_importer.es6.js

```diff
@@ -16,31 +16,72 @@
  */
 'use strict';
 
+require('./browser.es6');
+require('./web_interface.es6');
+require('./version_history.es6');
+
 foam.CLASS({
   name: 'ApiImporter',
   package: 'org.chromium.apis.web',
   documentation: `API Importer is a class that handles importing browserAPIs
     from webCatalog object to browserAPI DAO.`,
   requires: [
-    'org.chromium.apis.web.BrowserAPI',
+    'org.chromium.apis.web.Browser',
+    'org.chromium.apis.web.WebInterface',
+    'org.chromium.apis.web.BrowserWebInterfaceJunction',
+    'org.chromium.apis.web.VersionHistory',
     'foam.mlang.ExpressionsSingleton',
     'foam.dao.EasyDAO',
   ],
+  exports: [
+    'browserDAO',
+    'interfaceDAO',
+    'browserApiDAO as browserWebInterfaceJunctionDAO',
```

Comment:
1. In the api_matrix-test, the elements in junctionDAO are inserted manually. browser.interface.put() wasn't called. So it's browser.interface or interface.browser are null. 
      - It throws error when I call browser.interface.put if the code is outside a foam class. Seems it only looks for the relationshipJunctionDAO inside a foam class.

2. In web_apis-integration, the junctionDAO are inserted use browser.interface.put by apiImporter. It still fails even after I add 'export as ...'.